_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
```

Comment:
```suggestion
[our existing Interactive Documentation](https://help.apiary.io/tools/interactive-documentation/) (v3).
```

This sentence previously had a double “v3”, one inside the hyperlink and one just after in (brackets).